### ISSUE TITLE
New Exchange: KORBIT

### ISSUE THREAD

**Issue by jjhesk** [ccxt/ccxt#1212]
This exchange is came from Korea and it is a legitimate registered company,. API https://apidocs.korbit.co.kr/  this is urgent as this one of the large exchanges in the crypto world.
I think personal API is good enough.
fee: https://support.korbit.co.kr/customer/en/portal/articles/2745022

Korbit, Inc.376 Gangnam-daero 5th Fl., Gangnam-gu, Seoul, Korea  |  CEO Tony Lyu
Business Registration 220-88-61399  |  E-Commerce Registration 2013-Seoul Gangnam-01860

**Comment by jjhesk** [comment]
<img width="190" alt="2018-01-24 11 31 29" src="https://user-images.githubusercontent.com/629338/35319667-4b84ad02-00fa-11e8-92ce-a22fac96761f.png"> added image now

**Comment by mohnda** [comment]
@kroitor do you have plans to add this exchange anytime soon, please?

**Comment by kroitor** [comment]
@mohnda yep, we are working on it, however, I can't really promise on the ETA, as we tried to explain in the FAQ: https://github.com/ccxt/ccxt/wiki/FAQ. Will let you know as soon as it becomes available.

**Comment by mohnda** [comment]
@kroitor great, thank you!
